### PR TITLE
Repository setup

### PR DESCRIPTION
**Pull request by Fatima-hub333**
Create a New Repository according to Github workflow & run linters checks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,66 @@
...
+> I have to create a repository on my Github account, after that I have to create a branch and pull request, commit change 7 after all these things I've to delet that branch. The purpose of all this process is to get me familiar with github workflow.
+
+
+## Built With
+
+- Html & CSS
+
+## Live Demo (if available)
+
+[Live Demo Link](https://github.com/Fatima-hub333/Hello-World-Microverse-Project)
```
Comment (gemmen29) [inline README.md]: Kindly remove this section as there is no live demo, you add this only if there exists a live demo for the project

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+test.md
```
Comment (gemmen29) [inline .gitignore]: Kindly add a description for the PR with what you did exactly in that project in order for the PR to be professional 
you can edit this by pressing the three dots icon then press edit.
image

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @Fatima-hub333, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @Fatima-hub333, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes
_Check the comments under the review._

- Like the previous reviewer mentioned earlier in this [comment](https://github.com/Fatima-hub333/Hello_Microverse_Reapeated_Week/pull/1#discussion_r831277869) please make sure to add a description to your pull request as this is part of the requirements.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Fatima-hub333** [comment]
> Hi @Fatima-hub333,
> 
> Great work on making the changes clap You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
> 
> ## Suggested changes
> _Check the comments under the review._
> 
> * Like the previous reviewer mentioned earlier in this [comment](https://github.com/Fatima-hub333/Hello_Microverse_Reapeated_Week/pull/1#discussion_r831277869) please make sure to add a description to your pull request as this is part of the requirements.
> 
> Cheers and Happy coding!clapclapclap
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

How can I do this? I can't understand it from where I can be done with this? Can you help me with this?

**Comment by Lordkaito** [comment]
Hi Fatima, 

Let me try to help you.
Here, in the same page you are watching this message, scroll to the very top of it. You will see this:
Captura de pantalla 2022-03-21 173728

If you look closely, you will see, above of all your commit history, a section that says "No description provided". All you have to do is go to the right side of that box, find the three-dots icon, click it and then click edit. Now you will be able to edit the description of your PR, so all you need to do is add some description to it, and that's it!

Here's an image for reference of the three dots:
Captura de pantalla 2022-03-21 1737289

**Comment by Fatima-hub333** [comment]
> Hi Fatima,
> 
> Let me try to help you. Here, in the same page you are watching this message, scroll to the very top of it. You will see this: Captura de pantalla 2022-03-21 173728
> 
> If you look closely, you will see, above of all your commit history, a section that says "No description provided". All you have to do is go to the right side of that box, find the three-dots icon, click it and then click edit. Now you will be able to edit the description of your PR, so all you need to do is add some description to it, and that's it!
> 
> Here's an image for reference of the three dots: Captura de pantalla 2022-03-21 1737289

Oky Thank you. Kindly now have a look on iit:-)

**Review by gemmen29 — APPROVED** [review]
Hi @Fatima-hub333, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**